opensearch-operator/api/v1/opensearch_types.go

```diff
@@ -105,6 +105,15 @@ type BootstrapConfig struct {
 	Jvm          string                      `json:"jvm,omitempty"`
 }
 
+// +kubebuilder:validation:Enum=ClusterIP;NodePort;LoadBalancer
+type ServiceType string
+
+type ServiceSpec struct {
```

Comment:
To make it clearer I would prefer this to be named something like `DashboardsServiceSpec`.

Reply:
Yes makes perfect sense